uniapp-api-examples/App.vue

```diff
@@ -0,0 +1,17 @@
+<script>
```

Comment:
Could you move it to
`uniapp-api-examples/streaming-asr`?

We can add
`uniapp-api-examples/tts` and other examples later.